Make changes to datagrid

guergana opened the pull request:
- fixes #520

guergana commented:
@pdelboca @romicolman the metadata panel now shows below the screen, i think before it was absolutely positioned. I am not sure if this belongs to this ticket. Maybe that was introduced when we made changes from flex to grid system?

As you can see in the screenshot, now the user has to scroll to see the metadata panel. Before it was overlayed: 

![image](https://github.com/user-attachments/assets/e642e10e-80fd-4aff-bbce-542e8e26753c)

pdelboca commented:
@guergana we dicussed the issue with the metadata in the comments of this issue: https://github.com/okfn/opendataeditor/issues/520#issuecomment-2309832760

I think it would be nice to fix it here since we are implementing changes in the datagrid and the layout. Does it makes sense?

romicolman reviewed (commented):
Hi @guergana! I tested changes. Here are a few comments:

1) Just in case... are the font and size of the buttons **Metadata, source, errors report, undo and redo** and table cells aligned with the new design? 

2) I know we removed tooltips from the datagrid and changes were implemented OK, but  I can see them on the Publish and Save changes buttons now that we moved those features:

<img width="317" alt="Captura de pantalla 2024-09-04 a la(s) 9 27 01 a  m" src="https://github.com/user-attachments/assets/c614301a-a2b5-48a2-879e-6a72cbc3409c">

- I asked @faith to indicate if we will keep the red color (errors) as it is or if we need to adapt it ([see comment on the issue](https://github.com/okfn/opendataeditor/issues/520)). @Faithkenny can add a comment in the ticket so that Guergana can make changes if needed?

guergana commented:
> Hi @guergana! I tested changes. Here are a few comments:
> 
>     1. Just in case... are the font and size of the buttons **Metadata, source, errors report, undo and redo** and table cells aligned with the new design?
> 
>     2. I know we removed tooltips from the datagrid and changes were implemented OK, but  I can see them on the Publish and Save changes buttons now that we moved those features:
> 
> 
> <img alt="Captura de pantalla 2024-09-04 a la(s) 9 27 01 a  m" width="317" src="https://private-user-images.githubusercontent.com/20649846/364268391-c614301a-a2b5-48a2-879e-6a72cbc3409c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU3Mjg4NDMsIm5iZiI6MTcyNTcyODU0MywicGF0aCI6Ii8yMDY0OTg0Ni8zNjQyNjgzOTEtYzYxNDMwMWEtYTJiNS00OGEyLTg3OWUtNmE3MmNiYzM0MDljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA3VDE3MDIyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRlOTI3NDhmYWFhOTljNjc1MGVmNzFmOTAwZDYyODU3ZmNhYmZjOTY2ZGRjN2Y0YmJjY2Q4Njk2NGM0MjU3ZjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4sPI35ngBiBW-Yt6zsQs8ukyK8cOJH1QJS1hxpqa5r4">
> 
>     * I asked @faith to indicate if we will keep the red color (errors) as it is or if we need to adapt it ([see comment on the issue](https://github.com/okfn/opendataeditor/issues/520)). @Faithkenny can add a comment in the ticket so that Guergana can make changes if needed?

Hi @romicolman I tried to make a compromise between the buttons we currently have, as @pdelboca has been pointing out, and the design, since these buttons don't exist in the current code. I have removed the tooltips. :)

guergana commented:
> @guergana we dicussed the issue with the metadata in the comments of this issue: [#520 (comment)](https://github.com/okfn/opendataeditor/issues/520#issuecomment-2309832760)
> 
> I think it would be nice to fix it here since we are implementing changes in the datagrid and the layout. Does it makes sense?

Yes, sure @pdelboca

guergana commented:
> @guergana we dicussed the issue with the metadata in the comments of this issue: [#520 (comment)](https://github.com/okfn/opendataeditor/issues/520#issuecomment-2309832760)
> 
> I think it would be nice to fix it here since we are implementing changes in the datagrid and the layout. Does it makes sense?

@romicolman @pdelboca the panel position is fixed now.

@Faithkenny the color of the column with errors in the datagrid is `OKFNRed` now. 

Please review again everyone and let me know if further changes are needed.

romicolman commented:
Hi @guergana! I have just tested changes:

1) Te metadata and errors panel now look good.
2) Tooltips have been removed.
3) Faith added indications in terms of the red color to show errors. I still see the old one:

<img width="542" alt="Captura de pantalla 2024-09-09 a la(s) 10 29 16 a  m" src="https://github.com/user-attachments/assets/3b49b04e-c0b4-4466-baff-8d9110f2b218">

If this is not a small change, please, let's work on it after the pre-release.

guergana commented:
> Hi @guergana! I have just tested changes:
> 
>     1. Te metadata and errors panel now look good.
> 
>     2. Tooltips have been removed.
> 
>     3. Faith added indications in terms of the red color to show errors. I still see the old one:
> 
> 
> <img alt="Captura de pantalla 2024-09-09 a la(s) 10 29 16 a  m" width="542" src="https://private-user-images.githubusercontent.com/20649846/365572615-3b49b04e-c0b4-4466-baff-8d9110f2b218.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4NzMxMDksIm5iZiI6MTcyNTg3MjgwOSwicGF0aCI6Ii8yMDY0OTg0Ni8zNjU1NzI2MTUtM2I0OWIwNGUtYzBiNC00NDY2LWJhZmYtOGQ5MTEwZjJiMjE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA5VDA5MDY0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkMzFhY2YyOTMyOGEzODUwMWQ5NmUzNzFhMjRhODEwYWFlZWEwYjdlOTk4NWRlMmZjMDY5OWE3NTg1MjRiNmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.f3BudW3dHEiIvHEY-RyvtE0LrdkRFIU7eUgpxpAuk9Q">
> 
> If this is not a small change, please, let's work on it after the pre-release.

It's done, please check again.

romicolman commented:
I'm seeing the same color. Maybe the problem is connected to Mac?
